common/changes/@reshuffle/db-testsuite/configure-thoth-auth-token_2019-12-02-16-08.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@reshuffle/db-testsuite",
+      "comment": "Use useful API to set auth token in tests: a real token requires signing the appId, support that.",
+      "type": "minor"
```

Comment:
Note that need to upgrade to this version after.

Reply:
Thanks for making sure I don't forget!